app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Consider adding a comment here to explain the purpose of splitting the command string. This will improve code readability.